examples/Atlas/urdf/atlas_convex_hull.urdf

```diff
@@ -65,7 +65,7 @@
    <collision group="heel">
       <origin rpy="0 0 0" xyz="-0.082 0.0624435 -0.081119"/>
       <geometry>
-        <sphere radius="0"/>
+        <sphere radius="1e-7"/>
```

Comment:
if we really think that one should never use zero radius spheres, then I would prefer to leave it as zero in the urdfs, and parse it to 1e-7 in the urdf loader (perhaps using warnOnce).